src/composable/ConditionalOrder.ts

```diff
@@ -95,7 +96,7 @@ export abstract class ConditionalOrder<D, S> {
   assertIsValid(): void {
     const isValidResult = this.isValid()
     if (!isValidResult.isValid) {
-      throw new Error(`Invalid order: ${isValidResult.reason}`)
+      throw new Error(`Invalid order: ${(isValidResult as IsNotValid).reason}`)
```

Comment:
Probably this cast has to be done cause we didn't use type assertions. Not from this PR though

Reply:
It's not relevant anymore, deleted it